src/main/scala/nl/biopet/sbtbiopet/BiopetPlugin.scala

```diff
@@ -395,48 +384,51 @@ object BiopetPlugin extends AutoPlugin {
         runTest,
         setReleaseVersion,
         commitReleaseVersion,
-        tagRelease,
+        tagRelease
+      )
+    }
+  }
+
+  protected def biopetReleaseStepsSonatype: Def.Initialize[Seq[ReleaseStep]] = {
+    Def.setting[Seq[ReleaseStep]] {
+      Seq[ReleaseStep](
         releaseStepCommand(s"sonatypeOpen ${name.value}"),
         releaseStepCommand("publishSigned"),
-        releaseStepCommand("sonatypeReleaseAll"),
+        releaseStepCommand("sonatypeReleaseAll")
+      )
+    }
+  }
+
+  protected def biopetReleaseStepsGithub: Def.Initialize[Seq[ReleaseStep]] = {
+    Def.setting[Seq[ReleaseStep]] {
+      Seq[ReleaseStep](
         releaseStepCommand("ghpagesPushSite"),
         pushChanges,
-        releaseStepCommand("githubRelease"),
-        releaseStepCommand("git checkout develop"),
-        releaseStepCommand("git merge master"),
-        setNextVersion,
-        commitNextVersion,
-        pushChanges
+        releaseStepCommand("githubRelease")
       )
     }
   }
 
-  /*
-   * The ReleaseProcess for use with the sbt-release plugin
-   */
-  protected def biopetToolReleaseProcess: Def.Initialize[Seq[ReleaseStep]] = {
-    Def.setting {
+  protected def biopetReleaseStepsBioconda: Def.Initialize[Seq[ReleaseStep]] = {
+    Def.setting[Seq[ReleaseStep]] {
+      Seq[ReleaseStep](
+        releaseStepCommand("biocondaRelease")
```

Comment:
Maybe a method for 1 item it a bit to much?

Reply:
Only way to make it optional